### PR TITLE
Revert EditorList type changes

### PR DESCRIPTION
**Pull request by mdvictor**
Revert some of the changes introduced in https://github.com/grafana/grafana-experimental/pull/56 regarding `EditorList` type changes. 

This is needed to safely update the version of this package in grafana, which at the moment crashes, see https://github.com/grafana/grafana-experimental/issues/68.

### CODE REVIEW

**Review by jackw — APPROVED** [review]
In Zoltan's absence I'm happy to approve but you might want an approval from someone that knows the codebase better.

File: CHANGELOG.md
Comment (mdvictor) [reply CHANGELOG.md:5]: yes, it should, thank you!

**Comment by mdvictor** [comment]
@jackw I agree, I don't have much knowledge on this repo and am open to any recommendations on who to add in the reviewer list. It seems that many of the people that know the codebase are on leave and this PR needs to be merged asap to unblock our colleagues. Not sure who to connect with.

**Review by matyax — COMMENTED** [review]
Looks good to me 👍 !

**Review by asimpson — APPROVED** [review]
Looks good to me as well.

**Pull request merged**
